codepropertygraph/src/main/scala/io/shiftleft/SerializedCpg.scala

```diff
@@ -7,6 +7,7 @@ import java.net.{URI, URISyntaxException}
 import java.nio.file.{FileSystem, FileSystems, Files}
 import java.util
 
+@deprecated
```

Comment:
a comment re what to use instead is always welcome on deprecated messages.

Reply:
I accidentally added this change to the PR. Will address this in another one.